src/app/models/user.server.model.ts

```diff
@@ -75,8 +75,6 @@ const compileUserModel = (db: Mongoose) => {
         payment: Boolean,
         children: Boolean,
         postmanSms: Boolean,
-        // TODO: (MRF-email-notif) Remove betaFlag when MRF email notifications is out of beta
-        mrfEmailNotifications: Boolean,
```

Comment:
Can retain this flag as well. Same reasoning.

Reply:
retained in [d3a5fa7](https://github.com/opengovsg/FormSG/pull/7920/commits/d3a5fa713c5e9a821264a9fcc58718ae497926b0)